tests/python_simulators_test/battery_test.py

```diff
@@ -135,6 +173,44 @@ def test_SB_step(SB: SimpleBattery):
     assert SB.SOC == 0.1
     assert SB.P_charge == 0
 
+def test_SB_usage_calculations():
```

Comment:
Possibly label this a regression test? Something like `test_SB_usage_calculations_regression()`? Since (if I'm understanding this right), this test is more about checking that the outputs don't change than really checking for "correctness" of the result